Extension/package.json

```diff
@@ -5786,6 +5788,1419 @@
                         }
                     }
                 }
+            },
+            {
+                "type": "cpplldb",
+                "label": "C++ (LLDB-DAP)",
+                "languages": [
+                    "ada",
+                    "c",
+                    "cpp",
+                    "cuda-cpp",
+                    "rust"
+                ],
+                "variables": {
+                    "pickProcess": "extension.pickNativeProcess",
+                    "pickRemoteProcess": "extension.pickRemoteNativeProcess"
+                },
+                "configurationAttributes": {
+                    "launch": {
+                        "type": "object",
+                        "required": [
+                            "program"
+                        ],
+                        "properties": {
+                            "program": {
+                                "type": "string",
+                                "description": "%c_cpp.debuggers.program.description%",
+                                "default": "${workspaceFolder}/a.out"
+                            },
+                            "args": {
+                                "type": "array",
+                                "description": "%c_cpp.debuggers.args.description%",
+                                "items": {
+                                    "type": "string"
+                                },
+                                "default": []
+                            },
+                            "type": {
+                                "type": "string",
+                                "description": "%c_cpp.debuggers.cpplldb.type.description%",
```

Comment:
I'm seeing some warnings in the developer console:

![image](https://github.com/user-attachments/assets/f696fada-c233-48fc-9cf2-939a10f33a32)